test/users/engin/compiler/parentheseslessRecConst.chpl

```diff
@@ -0,0 +1,20 @@
+record independentRecord { var field = 5;} 
+
+var ir = new independentRecord;
+writeln(ir.field);
+//above lines compile with/without --verify
```

Comment:
I'd split these lines into a separate test.  I'd also be curious if others on the team are okay with being able to call `new typeName;` without parenthesis because I don't think this is a case we should support (parenthesesless calls should only be allowed on parenthesesless functions).  If we don't want to support it, that separate test should be a future calling for an error message on that code.